app/models.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from werkzeug.security import generate_password_hash, check_password_hash
+from itsdangerous import (TimedJSONWebSignatureSerializer
+                          as Serializer, BadSignature, SignatureExpired)
+from passlib.apps import custom_app_context as pwd_context
+
+app = Flask(__name__)
+app.config.from_object('config.DevelopmentConfig')
+db = SQLAlchemy(app)
+
+class UserModel(db.Model):
+    ___tablename__ = "user_model"
+
+    def __init__(self,username,password):
+        self.username = username
+        self.password = pwd_context.encrypt(password)
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(30))
+    password = db.Column(db.String)
+    bucketlists = db.relationship(
+        "Bucketlist", backref="user_model", lazy="dynamic")
+    bucketitems = db.relationship(
+        "BucketlistItem", backref="user_model", lazy="dynamic")
+
+
+    def verify_password(self, password):
+        return pwd_context.verify(password,self.password)
+
+
+    def generate_auth_token(self, expiration=3600):
+        s = Serializer(app.config['SECRET_KEY'], expires_in = expiration)
+        return s.dumps({"id": self.id})
+
+    @staticmethod
+    def verify_auth_token(token):
+        s = Serializer(app.config['SECRET_KEY'])
+        try:
+            data = s.loads(token)
+        except SignatureExpired:
+            return None
+        except BadSignature:
+            return None
+        return UserModel.query.get(data['id'])
+
+
+
+
+
```

Comment:
Resolved. Installed flake8 which checks fro pep8 and also used anaconda to autocheck pep8 errors